src/components/ComboBox/ComboBox.tsx

```diff
@@ -15,16 +15,6 @@ import {
 import { HeaderUI, InputUI, MenuUI, EmptyItemUI } from './ComboBox.css'
 import { COMPONENT_KEY } from './ComboBox.utils'
 
-export const shouldDropDirectionUpdate = (positionProps: any = {}) => {
```

Comment:
Removing this. This was causing the mechanism to be unreliable (for certain cases)